plugin-build/plugin/src/main/java/com/ncorti/ktfmt/gradle/KtfmtPluginUtils.kt

```diff
@@ -63,6 +63,49 @@ internal object KtfmtPluginUtils {
         }
     }
 
+    internal fun createScriptTasks(
+        project: Project,
+        projectDir: File,
+        topLevelFormat: TaskProvider<Task>,
+        topLevelCheck: TaskProvider<Task>,
+    ) {
+        val checkTaskName = "${TASK_NAME_CHECK}Script"
+        val formatTaskName = "${TASK_NAME_FORMAT}Script"
```

Comment:
nit: I'm wondering if those tasks should be pluralized instead:

```suggestion
        val checkTaskName = "${TASK_NAME_CHECK}Scripts"
        val formatTaskName = "${TASK_NAME_FORMAT}Scripts"
```

Reply:
Agreed and resolved in 4bd1fa6f9ddb4228c390763218eda1ffad842870